lib/auth0/api/v2/actions.rb

```diff
@@ -0,0 +1,201 @@
+module Auth0
+  module Api
+    module V2
+      # Methods to use the actions endpoints
+      module Actions
+        include Auth0::Mixins::Validation
+
+        attr_reader :actions_path
+
+        # Get all actions.
+        # @see https://auth0.com/docs/api/management/v2#!/Actions/get_actions
+        # @param trigger_id [string] An actions extensibility point.
+        # @param action_name [string] The name of the action to retrieve.
+        # @param deployed [boolean] filter to only retrieve actions that are deployed.
+        # @param per_page [integer] The amount of entries per page. Default: 50. Max value: 100.
+        # @param page [integer] The page number. Zero based.
+        # @param installed [boolean] When true, return only installed actions. When false, return only custom actions. Returns all actions by default.
+        # @return [json] Actions and pagination info
+        def actions(trigger_id: nil, action_name: nil, deployed: nil, per_page: nil, page: nil, installed: nil)
```

Comment:
Currently the endpoint does not support the `include_totals` parameter, but it will likely be added. Will it be easy to add support for that parameter in a non-breaking way here?

Reply:
yes, named parameters are basically destructuring a hash argument